### PR TITLE
[Channels] Add channel name to channels

### PR DESCRIPTION
**Pull request by sandeepkru**
Currently we are using everywhere slack's channelId which is randomly generated unique Id and not human friendly, so added channel name as well to the channels table so that we can use that instead of id in reports and on web.

### CODE REVIEW

**Comment by sandeepkru** [comment]
@rajatgoel thinking of having channel name is useful instead of using that unique channel ID everywhere, we can skip this if you feel like it might be useful, otherwise I can merge this change.

**Comment by rajatgoel** [comment]
Lets merge this.

**Review by rajatgoel — APPROVED** [review]

**Comment by rajatgoel** [comment]
Somehow we need to make all the web pages look similar (like same header at top, similar style).

File: internal/slack_integration/slack.go [hunk 1]
```diff
@@ -77,7 +77,16 @@ func (b *Integration) handleEventAPI(ctx context.Context, event slackevents.Even
...
+				// Get channel info before notifying
+				channelInfo, err := b.client.Client.GetConversationInfo(&slack.GetConversationInfoInput{
+					ChannelID: ev.Channel,
+				})
+				if err != nil {
+					log.Printf("Error getting channel info: %v", err)
+					return
+				}
+
+				err = b.bot.Notify(ctx, ev.Channel, channelInfo.Name)
```
Comment (rajatgoel) [inline internal/slack_integration/slack.go:89]: oh we should not do it everytime we get a message. probably add a channel without name and fill it async?

**Comment by sandeepkru** [comment]
> Somehow we need to make all the web pages look similar (like same header at top, similar style).

agree, let me also try that, this part with chatGPT also takes good amount of time :)

**Pull request merged**